src/load.c

```diff
@@ -39,14 +39,13 @@
 #include "loaders/loader.h"
 
 #ifndef LIBXMP_NO_DEPACKERS
-#if !defined(HAVE_POPEN) && defined(_WIN32)
-#include "win32/ptpopen.h"
-#define HAVE_POPEN 1
-#endif
 #if defined(__WATCOMC__)
 #define popen  _popen
 #define pclose _pclose
 #define HAVE_POPEN 1
+#elif !defined(HAVE_POPEN) && defined(_WIN32)
+#include "win32/ptpopen.h"
+#define HAVE_POPEN 1
 #endif
```

Comment:
Going to defer to @sezero for this pull request since I'm not familiar at all with Watcom.

This change is my only (minor) concern and it's more of a question—pt_popen.c should be safe (aside from reentrancy issues that will be addressed soon), whereas the Win32 `_popen` function is apparently questionable at best (source: [MSDN](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/popen-wpopen?view=msvc-160)) (#327). So, my question is, does pt_popen.c work for Watcom/Windows NT (in which case this change shouldn't be necessary)?

If pt_popen.c is fine for Windows NT, adding a `WIN32_OBJS` to the list of things `watcom-prepare` inserts into watcom.mif and appending that to `ALL_OBJS` in Makefile.nt might work.